libraries/oneview_helper.rb

```diff
@@ -102,5 +102,44 @@ def convert_keys(info, conversion_method)
       end
       support
     end
+
+    # Get the diff of the current resource state and the desired state
+    # @param [OneviewSDK::Resource] resource Resource containing current state
+    # @param [Hash] desired_data Desired state for the resource
+    # @return [String] Diff string (multi-line)
```

Comment:
I cannot see the expected behavior for *no diff* or *error* in the documentation comments.

Are those empty strings, nil...?